src/Prefabs/Weapons/Axe.et.meta

```diff
@@ -1,5 +1,5 @@
 MetaFileClass {
- Name "{BC5061EBB3284158}Prefabs/Weapons/Handguns/M9/Handgun_M9_empty.et"
```

Comment:
Please keep the empty weapon, add axe as separate prefab

Reply:
Yeah my bad, I thought that was my prefab because I was playing around with the M9, hopefully, that fixes it.